org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -451,6 +453,12 @@ enum ActionOrigin:
 enum Visibility:
     NONE | PRIVATE='private' | PUBLIC='public';
 
+enum BuiltinTrigger:
+    STARTUP='startup' | SHUTDOWN='shutdown' | RESET='reset';
+
+enum ModeTransition:
+    RESET='reset' | HISTORY='continue';
```

Comment:
I'm still a bit on the fence about the `continue` keyword. If refer to this a "history transition" and label it with an "H" in the diagram, then why don't we just use the `history` keyword and avoid all confusion?